code/modules/antagonists/brother/brother.dm

```diff
@@ -171,6 +169,34 @@
 	if (prob(10))
 		brothers_left += 1
 
+/datum/team/brother_team/add_member(datum/mind/new_member)
+	. = ..()
+	if (!new_member.has_antag_datum(/datum/antagonist/brother))
+		add_brother(new_member.current)
+
+/datum/team/brother_team/remove_member(datum/mind/member)
+	if (!(member in members))
+		return
+	. = ..()
+	member.remove_antag_datum(/datum/antagonist/brother)
+	if (!member.current)
+		return
+	for (var/datum/mind/brother_mind as anything in members)
+		to_chat(brother_mind, span_warning("[span_bold("[member.current.real_name]")] is no longer your Brother!"))
+	update_name()
+
+/// Adds a new brother to the team
+/datum/team/brother_team/proc/add_brother(mob/living/new_brother, source)
```

Comment:
what's the point of having return values if we don't listen to them? i like the idea of hinting at failure states/success states but I feel like it would be more useful to leverage the return value somehow.